WebCryptoAPI/generateKey/successes.js

```diff
@@ -65,6 +65,8 @@ function run_test(algorithmNames, slowTest) {
                     assert_goodCryptoKey(result, algorithm, extractable, usages, "secret");
                 }
             }, function(err) {
+                var supported = !err.toString().includes('not supported');
+                assert_precondition(supported, algorithm.name + ' not implemented');
```

Comment:
I'll move the bikeshedding here, since I guess this PR will have the docs and testharness.js changes. What will the resulting message in the report be?

Reply:
Ok, can actually answer this now that I've fixed it. It's the message in the second param to `assert_precondition`, e.g. 

![image](https://user-images.githubusercontent.com/777956/66340471-3a1adc00-e913-11e9-9ad3-21d009260594.png)

(bad alignment notwithstanding)